client/app/global-actions/global-actions.component.ts

```diff
@@ -0,0 +1,16 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'global-actions',
+  templateUrl: './global-actions.component.html',
+  styleUrls: ['./global-actions.component.css']
+})
+export class GlobalAction implements OnInit {
```

Comment:
What is this component for? I cant see it used anywhere

Reply:
Yes, it's being used in `app.component.html`. Notice the selector is plural. I'll fix it when I get home

Reply:
Yes, it's being used in `app.component.html`. Notice the selector is plural. I'll fix it when I get home

Reply:
Ok, and then you can merge